packages/arb-token-bridge-ui/src/components/common/WithdrawalCountdown.tsx

```diff
@@ -51,21 +74,24 @@ export function WithdrawalCountdown({
   createdAt: string | null
 }): JSX.Element {
   const {
-    l1: { network: l1Network }
+    l1: { network: l1Network },
+    l2: { network: l2Network }
   } = useNetworksAndSigners()
   const isLargeScreen = useMedia('(min-width: 1024px)')
+  const { isOrbitChain } = isNetwork(l2Network.id)
+  const baseChainId = isOrbitChain
```

Comment:
We use it for the 2nd time and it's a quite lengthy ternary. Maybe we can make it easier to read as suggested before, move it to helpers and reuse `getBaseChainId(chainId)` in both places?